src/routes/profile-route/profile-sidebar.jsx

```diff
@@ -42,6 +42,7 @@ export const ProfileSidebar = () => {
     setToken(null);
     setIsLoggedIn(false);
     client.resetStore();
+    location.reload();
```

Comment:
This was added to force a page reload on logout, as if a user were to logout without a page refresh, they would still be able to see the blog writer links until the page reloaded. These endpoints can still be navigated to by a user though, so perhaps it would be good to protect those routes from guests in a future ticket.